doc/tools/pkcs11-tool.1.xml

```diff
@@ -668,6 +695,24 @@
 			using the private key with ID <replaceable>ID</replaceable> and
 			using the RSA-PKCS mechanism:
 				<programlisting>pkcs11-tool --sign --id ID --mechanism RSA-PKCS --input-file data --output-file data.sig</programlisting>
+
+			To encrypt file using the AES key with ID 85 and using mechanism AES-CBC with padding:
+				<programlisting>
+pkcs11-tool --encrypt --id 85 -m AES-CBC-PAD \
+ --iv "00000000000000000000000000000000" \
+ -i file.txt -o encrypted_file.data
+				</programlisting>
+
+			Use the key with ID 22 and mechanism RSA-PKCS to unwrap key from file
+			<replaceable>aes_wrapped_key</replaceable> as AES key. This unwrapped
+			AES key becomes label <replaceable>unwrapeed-key</replaceable> and ID 90.
```

Comment:
I would rewrite this sentence to something like "After unwrapping the key is stored on token with a label ..."